---
layout: post
title: "New technique shows how cells interpret genetic information"
author: Tim Stephens
post_id: 2119
images:
  - file: http://www1.ucsc.edu/currents/02-03/art/splicing.02-08-05.224.jpg
    image_id: 2118
    caption: "DNA microarrays ('gene chips') enable researchers to monitor the RNA splicing process for all the genes in a cell simultaneously. The colored spots on this microarray show the results of an experiment comparing normal yeast with a yeast strain that carries a mutation in an essential RNA splicing factor. Photo: Tyson Clark"
---

[caption id="attachment_2118" align="alignnone" width="224"]<a href="http://localhost/mysite/wp-content/uploads/2002/08/splicing.02-08-05.224.jpg"><img class="size-full wp-image-2118" src="http://localhost/mysite/wp-content/uploads/2002/08/splicing.02-08-05.224.jpg" alt="DNA microarrays ("gene chips") enable researchers to monitor the RNA splicing process for all the genes in a cell simultaneously. The colored spots on this microarray show the results of an experiment comparing normal yeast with a yeast strain that carries a mutation in an essential RNA splicing factor. Photo: Tyson Clark" width="224" height="172" /></a>DNA microarrays ("gene chips") enable researchers to monitor the RNA splicing process for all the genes in a cell simultaneously. The colored spots on this microarray show the results of an experiment comparing normal yeast with a yeast strain that carries a mutation in an essential RNA splicing factor. Photo: Tyson Clark[/caption]
<p>
  A surprising amount of the DNA sequence in the genes of humans and other higher organisms ends up on the cutting-room floor, so to speak, spliced out by the cellular machinery that turns genetic code into functional proteins.
</p>
<p>
  Differences in the editing of genetic information may, in fact, be a significant source of genetic variability. UCSC researchers have now taken a big step toward understanding how this editing process (known as splicing) is regulated.<br>
</p>
<p>
  Using DNA microarrays (also called "gene chips"), the researchers are able to analyze the editing of all the genes in a cell simultaneously. This enables them to study how mutations or environmental perturbations affect the editing process.<br>
</p>
<p>
  "In the past, you had to analyze one gene at a time, but now we can look at all of the genes in parallel," said Manuel Ares, professor of molecular, cell, and developmental biology.<br>
</p>
<p>
  Master copies of all of an organism's genes are found on the chromosomes in every cell. When a gene is "expressed" or activated, its DNA sequence is copied into a "messenger RNA" molecule. The messenger RNA directs the production of a protein molecule, but not before a bit of cutting and splicing of the RNA code.<br>
</p>
<p>
  In the genes of higher organisms (as opposed to bacteria), the protein-coding information is broken up into fragments called "exons" that are separated by long sections of noncoding sequence called "introns." The splicing of pre-messenger RNA involves cutting out the introns and joining together the exons. The most striking thing about this process is that there are often several and sometimes many different ways to splice together the exons of a particular gene.<br>
</p>
<p>
  "It's estimated that more than 50 percent of human genes produce alternatively spliced messenger RNAs," Ares said. "To study this, we started by looking at yeast, because yeast genes have very few introns. The yeast genome is sort of a toy version of the bigger genomes of more complex organisms like humans."<br>
</p>
<p>
  Applying the microarray technique to yeast, Ares and his coworkers obtained the first genome-wide view of RNA splicing for any organism. They published their findings in the May 3 issue of the journal <i>Science.</i> Tyson Clark, a graduate student in molecular, cell, and developmental biology, is lead author of the paper. Coauthor Charles Sugnet is a graduate student in computational biology.<br>
  Ares's lab is also looking at RNA splicing in other organisms, including humans.
</p>
<p>
  For example, graduate student Valerie Welch has developed a set of microarrays to analyze changes in splicing as human cells in tissue culture progress from normal to cancerous states, and as cells are infected by viruses.<br>
</p>
<p>
  The idea that every gene acts by directing the production of a specific protein--known as the "one gene, one protein" hypothesis--was a Nobel Prize-winning milestone in the development of modern biology. But alternative splicing means that genes are actually much more complicated than that, and some genes can produce many proteins through alternative splicing. Most cases of alternative splicing involve just a few different versions, but one gene in the fruit fly apparently generates 38,000 different versions of its messenger RNA.<br>
  In humans, a good example is the gene for tropomyosin, a structural protein.
</p>
<p>
  Alternative splicing gives rise to five different versions of tropomyosin, which are produced in five different tissues in the body: skeletal muscle, smooth muscle, fibroblast, liver, and brain. Cells in each type of tissue splice the 11 exons of the tropomyosin gene differently to produce the different forms of the protein.<br>
</p>
<p>
  "The coding sequences of our genes are all broken up and spread out, and there is a whole cellular machinery involved in patching it together so that the code makes sense. This splicing process gives the cell the ability to try new combinatorial arrangements of information," Ares said. "You have all this information in the genome, but then the cell can interpret it in different ways. Our microarray technology allows us to access all that information in the cell at one time."<br>
</p>
<p>
  The technique depends on the recognition of the unique sequence created when two exons are spliced together. The initial step of identifying these splice-junction sequences involves comparing the genome sequence with the "expressed" sequence. To do this, Ares works with bioinformatics researchers led by David Haussler, professor of computer science and director of the Center for Biomolecular Science and Engineering. Sugnet performed this analysis for the yeast genome.<br>
</p>
<p>
  "He was able to identify the splicing events and, in each case, give us the coding sequence that spans the junction between the two exons. Then we created a synthetic DNA molecule that can recognize that sequence and spotted it onto the microarray," Ares said.<br>
</p>
<p>
  The microarrays for the yeast study consisted of glass slides with tiny spots of synthetic DNA laid out in a grid pattern. For every intron-containing gene in the yeast genome, the microarray included three synthetic DNA probes: splice-junction probes to detect spliced messenger RNA; intron probes to detect unspliced RNA; and exon probes to detect both spliced and unspliced RNA. RNA extracted from yeast cells and tagged with fluorescent labels was then placed on the microarray, allowing the RNA molecules present in the cells to bind to the specific DNA probes.<br>
</p>
<p>
  The intensity of the fluorescence at each spot serves as an indicator of the relative abundance of the corresponding RNAs. The microarrays are analyzed with an instrument that measures the fluorescence at each spot.<br>
</p>
<p>
  The researchers compared splicing patterns in normal and mutant strains of yeast. The mutations affected various factors known to be involved in the processing of messenger RNA molecules. In general, the results showed that the effects of RNA processing factors on splicing events depend on the particular intron involved.<br>
</p>
<p>
  "Each splicing event has its own character. When you alter the cell in some way, it's hard to predict the consequences for any particular intron," Ares said. "We hope to use our data to figure out what the rules are, why some splicing events depend more on certain parts of the RNA processing machinery than others."<br>
</p>
<p>
  Ares is already extending this work to human genes, starting with genes involved in cancer (tumor suppressor genes and oncogenes). Other researchers have reported that the splicing of some cancer genes changes as the cancer progresses.<br>
  Ultimately, Ares wants to understand the mechanisms that regulate RNA splicing in all cells. RNA splicing is basically an interpretive process--the genetic information contained in the genome is interpreted differently in different contexts.<br>
</p>
<p>
  "That interpretive process, and how it varies within a population, has really not been explored in any organism, including humans," Ares said. "I'd like to look at how much polymorphism [i.e., genetic variability] there is in the human population as far as how genes are interpreted."<br>
</p>
<p>
  This could be an important factor in understanding genetic diseases. The severity of certain genetic diseases, such as cystic fibrosis, varies tremendously depending on the patient's genetic background. In other words, the severity of the disease in two people with the same genetic defect might be quite different because of differences in other genes. Ares said differences in RNA splicing may have a lot to do with that.<br>
</p>
<p>
  "We know of cases where the way the disease gene is handled by the splicing machinery is different in different people," he said. "We have a lot to learn about this kind of variation."<br>
</p>
<p>
  <br>
  <br>

</p>
<p>

</p>
[source](http://www1.ucsc.edu/currents/02-03/08-05/splicing.html "Permalink to splicing")
